app/src/main/java/org/simple/clinic/summary/PatientSummaryEffectHandler.kt

```diff
@@ -295,13 +296,25 @@ class PatientSummaryEffectHandler @AssistedInject constructor(
           val phoneNumber = it.teleconsultationPhoneNumber
           val phoneNumbers = it.teleconsultationPhoneNumbers
 
-          if (phoneNumber.isNullOrBlank() || phoneNumbers.isNullOrEmpty()) {
-            TeleconsultInfo.MissingPhoneNumber
+          if (phoneNumbers.isNullOrEmpty()) {
+            teleconsultInfoFromPrimaryPhoneNumber(phoneNumber)
           } else {
-            TeleconsultInfo.Fetched(phoneNumber, phoneNumbers)
+            TeleconsultInfo.Fetched(phoneNumbers)
           }
```

Comment:
What do you think of:

```kotlin
when {
  phoneNumber.isNullOrBlank() && phoneNumbers.isNullOrEmpty() -> TeleconsultInfo.MissingPhoneNumber
  !phoneNumbers.isNullOrEmpty() -> TeleconsultInfo.Fetched(phoneNumbers)
  else -> TeleconsultInfo.Fetched(listOf(TeleconsultPhoneNumber(phoneNumber!!)))
}
```

Reply:
Yeah, we can go with that. Let me make the change.

Reply:
Done